OpenRA.Mods.TS/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -137,6 +157,41 @@ public void Run(ModData modData, string[] args)
 			{
 				// GetSection will throw when we run out of sections to import
 			}
+
+			metadata.Append("\tEditorTemplateOrder: ");
+
+			categories.Distinct();
+			metadata.Append(categories.First());
+			foreach (var category in categories.Skip(1))
+				metadata.Append(", " + category);
+
+			metadata.AppendLine();
+			metadata.AppendLine("\tSheetSize: 2048");
+			metadata.AppendLine("\tEnableDepth: true");
+			metadata.AppendLine();
+
+			metadata.AppendLine("Terrain:");
+			terrainTypes.Distinct();
```

Comment:
Distinct returns a new enumerable (it doesn't edit the collection in-place), so this does nothing.
This causes a whole pile of duplicated terrain types to be written when importing the TS temperate tileset.